### PR TITLE
Add BEP-20 BNB token list

### PR DESCRIPTION
**Pull request by hildobby**
Brief comments on the purpose of your changes:

Imported list of BEP-20 tokens from Dune v1 for BNB chain.

*For Dune Engine V2*
I've checked that:

* [X] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [X] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [X] if adding a new model, I added a test
* [X] the filename is unique and ends with .sql
* [X] each sql file is a select statement and has only one view, table or function defined  
* [X] column names are `lowercase_snake_cased`

When you are ready for a review, tag duneanalytics/data-experience. We will re-open your forked pull request as an internal pull request. Then your spells will run in dbt and the logs will be avaiable in Github Actions DBT Slim CI. This job will only run the models and tests changed by your PR compared to the production project.

### CODE REVIEW

File: models/tokens/bnb/tokens_bnb_schema.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+models:
+  - name: tokens_bnb_bep20
+    meta:
+      blockchain: bnb
+      sector: tokens
+      project: bep20
+      contibutors: hildobby
+    config:
+      tags: ['table', 'bep20', 'bnb']
```
Comment (jeff-dude) [inline models/tokens/bnb/tokens_bnb_schema.yml:11]: please add 'metadata' tag here, it'll help us do deployments quicker on hardcoded metadata views
reference to open PR on existing: https://github.com/duneanalytics/spellbook/pull/1438/files

Comment (jeff-dude) [reply models/tokens/bnb/tokens_bnb_schema.yml:11]: also, 'table' should be 'tokens' i believe

File: models/tokens/bnb/tokens_bnb_bep20.sql [hunk 2]
```diff
@@ -0,0 +1,5523 @@
+{{ config( alias='bep20')}}
+
+SELECT LOWER(contract_address) AS contract_address, symbol, decimals
+  FROM (VALUES 
+  
+('0x111111111117dc0aa78b770fa6a738034120c302', '1INCH', 18)
+,('0xfb6115445bff7b52feb98650c87f44907e58f802', 'AAVE', 18)
```
Comment (jeff-dude) [inline models/tokens/bnb/tokens_bnb_bep20.sql:7]: general comment: thank you for the left comma 🫡 
it also helps avoid the extra comma at the end when copy/pasting additional tokens and forgetting to remove it

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/tokens/bnb/tokens_bnb_bep20.sql (hunk 2 above)
Comment (hildobby) [reply models/tokens/bnb/tokens_bnb_bep20.sql:7]: i don't know you but i already like you

File: models/tokens/bnb/tokens_bnb_schema.yml (hunk 1 above)
Comment (hildobby) [reply models/tokens/bnb/tokens_bnb_schema.yml]: Done! ✅

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**